week14/solution.js

```diff
@@ -0,0 +1,49 @@
+function bfs(place) {
+    const places = []
+    for (const element of place) {
+        places.push(element.split(''));
+    }
+
+    const queue = [];
+    for (let i = 0; i < 5; i++) {
+        for (let j = 0; j < 5; j++) {
+            if (places[i][j] === 'P') {
+                
+                queue.push([i, j, 0]);
+                  const visited = [[false], [false], [false], [false], [false]];
```

Comment:
```suggestion
                  const visited = new Array(5).fill(new Array(5).fill(false));
```
로 하는것이 조금 더 안전하지 않을까요?

Reply:
넵 좋은것 같습니다!